app/src/main/res/layout/fragment_login.xml

```diff
@@ -76,7 +76,7 @@
                     android:id="@+id/loginButton"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:enabled="false"
+                    android:enabled="true"
```

Comment:
This should only enable when everything is correct

Reply:
Ok @iamareebjamal sir, will do that and will update the pull request with necessary changes.

Reply:
Thanks @iamareebjamal will use textwatcher and will update the pull request with necessary changes.